frontend/src/components/BottomBanner/index.tsx

```diff
@@ -75,10 +111,13 @@ export default function BottomBanner({
     setNewsletterModalIsOpen(!newsletterModalIsOpen);
   }
 
-  // eslint-disable-next-line sonarjs/cognitive-complexity
+  /**
+   * Reads a query parameter from the URL to auto open the newsletter signup modal
+   * Allows sharing of a URL to lead directly to the newsletter signup, specifically for conferences
+   */
   useEffect(() => {
-    // Reads a query parameter from the URL to auto open the newsletter signup modal
-    // Allows sharing of a URL to lead directly to the newsletter signup, specifically for conferences
+    if (!isHubSpotReady) return;
```

Comment:
guard clause for readability - a long block nested in a condition caught me off guard